### PR TITLE
fastboot: flash the partition after downloading the file

### PR DESCRIPTION
**Pull request by aleksander0m**
This performs the partition flashing operation, which was missing after the file download.

### CODE REVIEW

File: plugins/fastboot/fu-fastboot-device.c [hunk 1]
```diff
@@ -283,6 +284,14 @@ fu_fastboot_device_flash (FuDevice *device,
...
 				      FU_FASTBOOT_DEVICE_READ_FLAG_STATUS_POLL, error))
 		return FALSE;
+
+	/* actual partition flashing after download */
+	if (!fu_fastboot_device_writestr (device, flash_cmd, error))
+		return FALSE;
+	if (!fu_fastboot_device_read (device, NULL,
+				      FU_FASTBOOT_DEVICE_READ_FLAG_STATUS_POLL, error))
+		return FALSE;
+
```
Comment (superm1) [inline plugins/fastboot/fu-fastboot-device.c]: @hughsie Just as a general observation (and not that I think it should block this PR) it seems that these functions (and some others in `*_device_flash`) are using effectively what's in `fu_fastboot_device_cmd` but with a polling flag.

is it worth adding an argument that causes that behavior to `fu_fastboot_device_cmd`?

Comment (hughsie) [reply plugins/fastboot/fu-fastboot-device.c]: I've done this as a follow up commit -- comments welcome.

**Comment by hughsie** [comment]
@aleksander0m if that looks okay to you with my modifications please merge.

**Comment by hughsie** [comment]
/me does that so it makes the release, I hope that's okay.

**Pull request merged**